README.md

```diff
@@ -180,3 +201,64 @@ produce early errors because of recursive application of the first rule.
 - If Type(_targetName_) is not `String`, then let _targetName_ be the empty string.
 - Let _status_ be ? SetFunctionName(_F_, _targetName_, `"bound"`).
 - Return _F_.
+
+----
+
+### Bound Constructor Exotic Objects
+
+A bound constructor is an exotic object that wraps a function object with a [[Construct]] internal method. A bound constructor is callable (it has a [[Call]] internal method) and constructible (it has a [[Construct]] internal method). Calling a bound constructor generally results in a call of its wrapped function's constructor.
+
+Bound constructor objects do not have the internal slots of ECMAScript function objects defined in Table 27. Instead they have a single slot, [[BoundTargetFunction]], which is a Callable Object set to the wrapped function object.
+
+Unlike ECMAScript function objects, bound constructor objects do not use an alternative definition of the [[GetOwnProperty]] internal methods, and unlike bound function objects, they are only callable and do not have a prototype. Bound constructor objects provide all of the essential internal methods as specified in 9.1. However, they use the following definitions for the essential internal methods of constructor objects.
```

Comment:
That "*+Unlike ECMAScript function objects, bound constructor objects do not use an alternative definition of the [[GetOwnProperty]] internal methods*" part [should probably omitted](https://bugs.ecmascript.org/show_bug.cgi?id=4549).

Also I'd recommend to be explicit about "*do not have a `.prototype` property*" to avoid confusion.